pom.xml

```diff
@@ -264,28 +212,27 @@
           </execution>
         </executions>
       </plugin>
-<!--
+
       <plugin>
         <groupId>org.openapitools</groupId>
         <artifactId>openapi-generator-maven-plugin</artifactId>
         <version>${openapi-generator.version}</version>
         <executions>
           <execution>
-            <id>inn-reach</id>
+            <id>DTO Generation</id>
             <goals>
               <goal>generate</goal>
             </goals>
             <configuration>
-              <inputSpec>${i2nreach.yaml.file}</inputSpec>
+              <inputSpec>${openapi.input.file}</inputSpec>
               <output>${project.build.directory}/generated-sources</output>
               <generatorName>spring</generatorName>
-              <modelPackage>${project.groupId}.edge.i2nreach.domain.dto</modelPackage>
-              <apiPackage>${project.groupId}.edge.i2nreach.rest.resource</apiPackage>
+              <modelPackage>${project.groupId}.edge.dto</modelPackage>
+              <apiPackage>${project.groupId}.edge.rest.resource</apiPackage>
               <generateApis>true</generateApis>
               <generateApiTests>true</generateApiTests>
               <generateApiDocumentation>true</generateApiDocumentation>
               <generateModels>true</generateModels>
```

Comment:
We states here about generating models but removed list of models to generate. Can we skip their listing here?